### PR TITLE
update: contactUs page

### PR DESCRIPTION
**Pull request by HeemahMuslad**

### CODE REVIEW

File: src/components/Recaptcha.jsx [hunk 1]
```diff
@@ -5,11 +5,11 @@ function Recaptcha({ onToken }) {
 
   useEffect(() => {
     const script = document.createElement("script");
-    script.src = "https://www.google.com/recaptcha/api.js?render=6LdgNhIqAAAAAHaQlxrKbWOSCh79HjlWz8JfCEOE";
+    script.src = "https://www.google.com/recaptcha/api.js?render=6LeMShEqAAAAANL4rT2yerYM70w-yM_HPi4TvrEP";
     script.addEventListener("load", () => {
       window.grecaptcha.ready(() => {
         window.grecaptcha
-          .execute("6LdgNhIqAAAAAHaQlxrKbWOSCh79HjlWz8JfCEOE", { action: "submit" })
+          .execute("6LeMShEqAAAAANL4rT2yerYM70w-yM_HPi4TvrEP", { action: "submit" })
```
Comment (Lezette) [inline src/components/Recaptcha.jsx:12]: Add the key to an env file and use it. This is a key that should not be exposed to everyone

File: src/components/Recaptcha.jsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ function Recaptcha({ onToken }) {
     };
   }, [onToken]);
 
-  return <div ref={recaptchaRef} className="g-recaptcha" data-sitekey="6LdgNhIqAAAAAHaQlxrKbWOSCh79HjlWz8JfCEOE" data-size="invisible"></div>;
+  return <div ref={recaptchaRef} className="g-recaptcha" data-sitekey="6LeMShEqAAAAANL4rT2yerYM70w-yM_HPi4TvrEP" data-size="invisible"></div>;
```
Comment (Lezette) [inline src/components/Recaptcha.jsx:25]: Same thing here

File: src/components/contact-us/index.jsx [hunk 3]
```diff
@@ -50,7 +50,7 @@ const ContactUsComponent = () => {
   const onsubmit = (data) => {
     window.grecaptcha.ready(() => {
       window.grecaptcha
-        .execute("6LdgNhIqAAAAAHaQlxrKbWOSCh79HjlWz8JfCEOE")
+        .execute("6LeMShEqAAAAANL4rT2yerYM70w-yM_HPi4TvrEP")
```
Comment (Lezette) [inline src/components/contact-us/index.jsx:53]: And here

File: src/components/Recaptcha.jsx [hunk 4]
```diff
@@ -5,11 +5,11 @@ function Recaptcha({ onToken }) {
 
   useEffect(() => {
     const script = document.createElement("script");
-    script.src = "https://www.google.com/recaptcha/api.js?render=6LdgNhIqAAAAAHaQlxrKbWOSCh79HjlWz8JfCEOE";
+    script.src = "https://www.google.com/recaptcha/api.js?render=6LeMShEqAAAAANL4rT2yerYM70w-yM_HPi4TvrEP";
```
Comment (Lezette) [inline src/components/Recaptcha.jsx:8]: Remove the code, add it to the .env file and call the env variable here to use it

**Review by Lezette — CHANGES_REQUESTED** [review]
Codes and similar items are meant to be secret keys this should not be used inside the code, add it to an env file then use the variable.

When you're done, reach out to Eniola and inform her that they need to update the env file from the deployed page as well.

**Comment by HeemahMuslad** [comment]
https://www.loom.com/share/f1cd855bd3bc4693b072afd8222f6bdc

**Review by Lezette — APPROVED** [review]
LGTM

**Review by Lezette — APPROVED** [review]

**Review by EseAlli — APPROVED** [review]

**Pull request merged**